### DISCUSSION TITLE
Hi, comment here to send your github user

### DISCUSSION THREAD

**Discussion by pablo-icm** [marbits-icm/marbits-public#6] [General]
Hi everyone. I'm editing this thread to say that you will be invited to the repo `marbits-public` with "Write" role. This means that you could push content to the repo (mainly documentation, scripts, etc) so as this starts to work more like a community. Please, be nice and respectful (as you always are, BTW). 

Remember that you are suscribed to al threads where you are mentioned or have contributed and will receive a mail when there's an update (likely you are reding this in your mail). You can manage your notifications preferences for each thread or in general in your settings.

**Comment by xabilopalf** [comment]
xabilopalf

**Comment by ClaraRuizG** [comment]
ClaraRuizG

**Comment by CaterinaRG** [comment]
CaterinaRG

**Comment by clagovaldivia** [comment]
clagovaldivia

**Comment by felipehcoutinho** [comment]
✌🏽

**Comment by pjmramond** [comment]
pjmramond

**Comment by Skogstokigg** [comment]
Skogstokigg

**Comment by Ulises-Lora** [comment]
Ulises-Lora

**Comment by Skogstokigg** [comment]
cergare

**Comment by jeffmangot** [comment]
jeffmangot

**Comment by aleixop** [comment]
aleixop

**Comment by martaroyo** [comment]
martaroyo

**Comment by mvquiroga** [comment]
mvquiroga

**Comment by NataliaTimoneda** [comment]
NataliaTimoneda

**Comment by adriaaula** [comment]
adriaaula

I have to add that unless you do this, for some files, you cannot even open them. I am unable right now for example to read the doc on how to connect [through VPN](https://github.com/marbits-icm/marbits-public/wiki/tunnel)

**Comment by ehdelamo** [comment]
ehdelamo
